### DISCUSSION TITLE
Any way to send a sliced file?

### DISCUSSION THREAD

**Discussion by PrintsLeo3d** [nlef/moonraker-telegram-bot#226] [Q&A] [answered]
Hi all, and thanks for your time. I've been loving how simple the telegram-bot has been for my 3d printing needs, but I have been wondering is there anyway to send a sliced file through the telegram system, having it upload to the printer? The ease of picking and printing an already sliced and uploaded file is amazing! I am just trying to figure out if anyone knows, or can think of, a way to upload a file remotely as well. -Leo

**Comment by aka13-404** [comment, marked as answer]
Yeah, just send the bot a zip, or a gcode to the chat, and it will do exactly as you wish.

**Reply by PrintsLeo3d** [reply]
I cannot believe it is that easy!! Total mind blow. Thanks for the quick response, I am going to try it now!
